Jenkinsfile

```diff
@@ -0,0 +1,3 @@
+common {
+    cron = null
```

Comment:
what's this for?

Reply:
this file is used by jenkins to build the artifacts. We don't need this being triggered on a cron schedule so setting cron to null.